growattServer/__init__.py

```diff
@@ -116,12 +116,15 @@ def login(self, username, password, is_password_hashed=False):
             'userName': username,
             'password': password
         })
-        data = json.loads(response.content.decode('utf-8'))['back']
-        if data['success']:
-            data.update({
-                'userId': data['user']['id'],
-                'userLevel': data['user']['rightlevel']
-            })
+        try:
+            data = json.loads(response.content.decode('utf-8'))['back']
+            if data['success']:
+                data.update({
+                    'userId': data['user']['id'],
+                    'userLevel': data['user']['rightlevel']
+                })
+        except:
+            data = {'success': False}
```

Comment:
Is it maybe possible to attach the exception message to `msg` here?
Just as extra info for whoever is using this and running into an error